docs/cloud-native-security/cspm-findings.asciidoc

```diff
@@ -19,10 +19,12 @@ CSPM findings indicate whether a given resource passed or failed evaluation agai
 By default, the Findings page lists all findings, without grouping or filtering.
 
 [discrete]
-=== Group findings by resource
+=== Group findings 
+
+. Click **Group findings by:** to view a list of fields by which you can group findings. Select one of the suggested group-by fields, or click **Custom field** to choose your own group-by field. 
+. When grouping is enabled, click a group to expand it and view all findings within that group.
```

Comment:
Optional edit to language to avoid using ableist terms.

```suggestion
. When grouping is turned on, click a group to expand it and examine all findings within that group.
```